bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -116,39 +119,41 @@ val copyResources by tasks.register<Copy>("copyResources") {
 val downloadJnaLibs by tasks.register<DefaultTask>("downloadJnaLibs")
 val revision: String = System.getProperty("gitRevision", if (gitVersion == version) gitVersion else gitHash)
 
-data class LibMongoCryptS3Data(val evergreenName: String, val classifier: String, val osArch: String) {
+data class LibMongoCryptS3Data(val evergreenName: String, val osArch: String) {
```

Comment:
`classifier` was never fully removed.  Now it is.

Reply:
Ack